src/js/components/HealthBar.js

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import React from 'react';
+import {Tooltip} from 'reactjs-components';
+
+import HealthBarMap from '../constants/HealthBar';
+import StatusBar from './StatusBar';
+
+class HealthBar extends React.Component {
+  renderToolTip() {
+    let {tasks, instancesCount} = this.props;
+
+    tasks = Object.keys(tasks).filter(function (task) {
+      return tasks[task] !== 0 && task !== 'tasksRunning';
+    }).map(function (task, index) {
+      let percentage = parseInt(tasks[task] / instancesCount * 100, 10);
+
+      let classSet = classNames(HealthBarMap[task].cssName, 'dot icon');
+
+      return (
+        <div key={index}>
+          <span className={classSet}></span>
+          {` ${tasks[task]} ${HealthBarMap[task].label} (${percentage} %)`}
+        </div>
+      );
+    }).filter(function (item) {
```

Comment:
why do we need this filter? won't the previous map ensure that it returns something?